tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -56,41 +44,20 @@ public function testGetPackageInfo()
         ], $output);
     }
 
-    public function testGetSupportedPackagesEchosErrors()
-    {
-        $supportedAddonsLoader = $this->mockSupportedAddonsLoader();
-        $moduleHealthLoader = $this->mockModuleHealthLoader();
-
-        $supportedAddonsLoader->expects($this->once())
-            ->method('getAddonNames')
-            ->will($this->throwException(new RuntimeException('A test message')));
-
-        /** @var UpdatePackageInfoTask $task */
-        $task = UpdatePackageInfoTask::create();
-        $task->setSupportedAddonsLoader($supportedAddonsLoader);
-        $task->setModuleHealthLoader($moduleHealthLoader);
-
-        ob_start();
-        $task->getSupportedPackages();
-        $output = ob_get_clean();
-
-        $this->assertStringContainsString('A test message', $output);
-    }
```

Comment:
There's not a clean way to mock this anymore - but it's also very clear and deliberate code that causes this to happen, we'd have to intentionally change it to not echo the exception in order to introduce a regression here.